sdk/spring-experimental/spring-cloud-azure-native-configuration/CHANGELOG.md

```diff
@@ -4,9 +4,6 @@
 
 ### Features Added
 
-<!-- 
-NOTE: This changelog entry should be added back to Features Added section of the 'spring/CHANGELOG.md' when the native configuration is ready to release the beta version, and referenced here.
--->
 - Add library `spring-cloud-azure-native-configuration` to support `Spring Native` for compiling Spring Cloud Azure applications to native executables [#28053](https://github.com/Azure/azure-sdk-for-java/issues/28053), [#28158](https://github.com/Azure/azure-sdk-for-java/pull/28158).
```

Comment:
`to native executables` seems not needed. It is not correct in grammar.

Reply:
how about checking this https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#overview.